accepted/PSR-13-links.md

```diff
@@ -272,7 +272,7 @@ interface EvolvableLinkInterface extends LinkInterface
 }
 ~~~
 
-#### 3.2 `Psr\Link\LinkProviderInterface`
+### 3.3 `Psr\Link\LinkProviderInterface`
```

Comment:
As PSR-13 is already accepted, people referring to these numbers; therefore, it would be unfortunate to change these in my opinion.